fastlane/jetpack_metadata/ru/subtitle.txt

```diff
@@ -1 +0,0 @@
-Быстрый и безопасный WordPress
```

Comment:
⚠️  Remember to check that one before final release too (just like you said for `es-ES` above)